nncf/openvino/graph/node_utils.py

```diff
@@ -337,3 +337,47 @@ def get_weight_channel_axes(node: NNCFNode, weights_port_id: int) -> List[int]:
             channel_axes.append(matmul_channel_axis)
 
     return channel_axes
+
+
+def get_activation_channel_axis(node: NNCFNode) -> int:
+    """
+    Returns axis number of the activation tensor which correspond to it channel.
+
+    :param node: NNCFNode instance.
+    :return: Channel axis number.
+    """
+    channel_axis = node.metatype.input_channel_axis if node.metatype.input_channel_axis is not None else 1
+
+    if (
+        node.layer_attributes is not None
+        and node.layer_attributes.input_attributes is not None
+        and "transpose" in node.layer_attributes.input_attributes
+        and node.layer_attributes.input_attributes["transpose"]
+    ):
+        channel_axis -= 1
```

Comment:
Are you sure that this transpose operation is correct? General formula should be like this I suppose - https://github.com/openvinotoolkit/nncf/blob/develop/nncf/onnx/graph/node_utils.py#L117

Reply:
See - https://docs.openvino.ai/2023.0/openvino_docs_ops_matrix_MatMul_1.html.
It's correct.